### DISCUSSION TITLE
Gopro 11  Labs settings

### DISCUSSION THREAD

**Discussion by BradACI** [gopro/labs#286] [General]
https://youtu.be/0981rR0qheI

**Comment by dnewman-gpsw** [comment]
BITR =1 is not a thing, well least 1mb/s is not recommended (I'm not sure if this is typo, or the camera is ignoring the property.)  BITR=170 is better.

**Reply by BradACI** [reply]
Yep. Typo. Thanks for catching that.

**Reply by dnewman-gpsw** [reply]
Hi Brad, welcome to having fun with Labs.

If you ever want to read back the Labs setting applied to a particular MP4 use  https://gopro.github.io/labs/control/metadata/

example (look at the very bottom for User Metadata, this is the Labs settings): 
![image](https://user-images.githubusercontent.com/10856084/227733054-c5033dfc-b86b-4745-9df6-b3a95086aa10.png)

**Comment by andrerav** [comment]
Nice results. I haven't played with the DAMP setting myself yet. Should probably try out both faster and slower dampening since I often ride through rapidly changing lighting conditions. Not sure what would be most beneficial. And, with my current goto exposure settings (LOGB=400 and EV=-2) the exposure seems to be quite "calm" either way, so DAMP might not make such a big difference regardless.

**Reply by BradACI** [reply]
DAMP .1 better than maxing it out.the difference is very, very noticeable. I am going to keep this setting. It would have been great if they had this setting for the Gopro Max.
